app/src/main/java/com/example/sharingang/EditItemFragment.kt

```diff
@@ -15,8 +20,19 @@ import com.example.sharingang.items.ItemsViewModel
 class EditItemFragment : Fragment() {
 
     private val viewModel: ItemsViewModel by activityViewModels()
+
+    private lateinit var contextActivity: MainActivity
     private lateinit var existingItem: Item
 
+    private var imageUri = MutableLiveData<Uri>()

```

Comment:
I don't think this should be in the Fragment directly, because I don't think it gets persisted when for example, the screen rotates, restarting the fragment.